src/SIM/ModelGenerator.C

```diff
@@ -62,32 +64,29 @@ std::string DefaultGeometry1D::createG2 (int nsd) const
     for (d = 1; d < nsd; d++)
       g2.append(" 0.0");
   }
-  if (rational)
-    g2.append(" 1.0");
-  g2.append("\n");
+  g2.append(rational ? " 1.0\n" : "\n");
   if (utl::getAttribute(geo,"X1",XYZ))
   {
-    IFEM::cout <<"\tX1 = "<< XYZ << std::endl;
+    IFEM::cout <<"\n\tX1 = "<< XYZ;
     g2.append(XYZ);
   }
   else
   {
     XYZ = "1.0";
     if (utl::getAttribute(geo,"L",XYZ))
-      IFEM::cout <<"  Length scale: "<< XYZ << std::endl;
+      IFEM::cout <<"\n\tLength: "<< XYZ;
```

Comment:
sure, fixed in ec496f045b9f8a8d7a64c901a6c86d48c778a3a8. Think I got all of them.